<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>zControllerEditor</class>
 <widget class="QDialog" name="zControllerEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>729</width>
    <height>566</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Edit Controllers</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="0">
    <widget class="QWidget" name="mainBox" native="true">
     <layout class="QVBoxLayout" name="mainLayout">
      <item>
       <layout class="QHBoxLayout" name="buttonLayout">
        <item>
         <widget class="QToolButton" name="createControllerButton">
          <property name="text">
           <string>Create controller</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="createControllerGroupButton">
          <property name="text">
           <string>Group</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="deleteButton">
          <property name="text">
           <string>Delete</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="ControllerTreeWidget" name="controllerTree">
        <property name="showDropIndicator" stdset="0">
         <bool>true</bool>
        </property>
        <attribute name="headerVisible">
         <bool>false</bool>
        </attribute>
        <column>
         <property name="text">
          <string notr="true">1</string>
         </property>
        </column>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ControllerTreeWidget</class>
   <extends>QTreeWidget</extends>
   <header>zMayaTools/qt_widgets/controller_tree_widget.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../qt_resources/resources.qrc"/>
 </resources>
 <connections/>
</ui>
